client/src/components/pages/Modeler/Modeler.jsx

```diff
@@ -34,6 +30,7 @@ const Modeler = (match) => {
    */
   useEffect(() => {
     setRobotId(robotId);
+    initSessionStorage('idCounter', JSON.stringify('541'));
```

Comment:
Where is this magic number coming from? What does this have to do with the id?

Reply:
This is just the initial value for the ID of the first element. Since we are not using an ID package, I have set this Magic Number.